src/main/java/dev/gigafyde/apollo/commands/music/Lyrics.java

```diff
@@ -0,0 +1,165 @@
+package dev.gigafyde.apollo.commands.music;
+
+import com.sedmelluq.discord.lavaplayer.track.AudioTrack;
+import dev.gigafyde.apollo.core.command.Command;
+import dev.gigafyde.apollo.core.command.CommandEvent;
+import dev.gigafyde.apollo.core.command.SlashEvent;
+import dev.gigafyde.apollo.utils.SongUtils;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.interactions.InteractionHook;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.awt.*;
+import java.io.IOException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Lyrics extends Command {
+
+    private static final Logger log = LoggerFactory.getLogger(Lyrics.class);
+    public static String web = System.getenv("LYRICS_WEB_SERVER");
+    public static String key = System.getenv("LYRICS_API_KEY");
```

Comment:
This can be simplified by calling Main.LYRICS_WEB_SERVER and Main.LYRICS_API_KEY